modules/ROOT/pages/relnotes-nodejs-sdk.adoc

```diff
@@ -28,6 +28,44 @@ specify the path to the install root on the command line:
 npm install couchbase --couchbase-root=/opt/libcouchbase-2.9.3
 ----
 
+== Version 2.6.0 (27 July 2018)
+
+Version 2.6.0 is the first general availability release of the
+2.6 series. It adds support for the upcoming Couchbase Server 6.0.
+
+[source,bash]
+----
+$ npm install couchbase@2.6.0
+----
+
+http://docs.couchbase.com/sdk-api/couchbase-node-client-2.6.0/[API Reference]
+
+=== New Features
+
+* http://issues.couchbase.com/browse/JSCBC-497[JSCBC-497]:
+Added expanded support for Couchbase Analytics.
+* http://issues.couchbase.com/browse/JSCBC-508[JSCBC-508]:
+Added helper for specifying custom N1QL query options.
```

Comment:
Can we say something like "See blah in the API reference for further details." ?